### ISSUE TITLE
Example 'rfft2.rs' fails on my machine

### ISSUE THREAD

**Issue by skailasa** [preiter93/ndrustfft#7]
This example with real 2 complex FFTs /inverse FFTs fails on my machine. (I'm using an M1 MB pro, though not sure if hardware impacts fft algs in rustfft).

The forward transform seems to work, but the inverse transform returns false results. I've pasted the results below.

Do you have any ideas for where this bug could be coming from?

This is a great package by the way, and really helps my usecase!

```
v =[[1.0, 2.0, 3.0],
 [4.0, 5.0, 6.0],
 [7.0, 8.0, 9.0]], shape=[3, 3], strides=[3, 1], layout=Cc (0x5), const ndim=2 
 
 vnew = [[1.5000000000000029, 3.0000000000000013, 4.500000000000002],
 [5.999999999999999, 7.499999999999995, 8.999999999999995],
 [10.5, 11.999999999999995, 13.499999999999995]], shape=[3, 3], strides=[3, 1], layout=Cc (0x5), const ndim=2
```

**Comment by preiter93** [comment]
Can you try v3.0? 
I think there is something wrong with the default normalization in irfft. 

Alternatively you cann use a custom normalization, there is an example too.

Unfortunately I can check earliest on friday.

**Comment by skailasa** [comment]
No that's alright I can wait til Friday. Thanks for this.

I have a couple of questions though.

1. For the parallel methods what are you parallelising over?
2. Which algorithm are you using as the default for the FFT, just the default one?
3. I assume neon/sse are being enabled by default?

**Comment by preiter93** [comment]
Maybe something like this works too

```
   let my_norm = |data: &mut [Complex<f64>]| {
        let n = T::one() / T::from_usize(ny).unwrap();
        for d in data.iter_mut() {
            d.re = d.re * n;
            d.im = d.im * n;
        }
    };
    let mut handler_ax1 = R2cFftHandler::<f64>::new(ny).normalization(Normalization::Custom(my_norm) as fn(&mut [Complex<f64>]));
 ```
(I am not on my laptop so I havent tested it)

**Comment by preiter93** [comment]
Algorithm is taken care of by rustfft, and neon should be enabled by default when I understand the Rustfft doc correctly.

Its parallelized over "lanes" using ndarrays parallel iterator. So if you have a 10x6 array and you do a fft with 2 processors over axis 1, each processor does 5 ffts of lengtjh 6

**Comment by preiter93** [comment]
Fixed in v0.4.1.
